app/components/TaskList/index.tsx

```diff
@@ -256,18 +256,37 @@ const TaskList: React.FC<Props> = ({ projectRevision, mode }) => {
           />
         </TaskListSection>
 
-        {/* Annual Reports Section */}
+        {/* Emissions Intensity Report */}
         <TaskListSection
           defaultExpandedState={
             currentStep === "6" ||
-            annualReportsStatus === ATTENTION_REQUIRED_STATUS
+            quarterlyReportsStatus === ATTENTION_REQUIRED_STATUS
           }
           listItemNumber="6"
-          listItemName="Annual Reports"
+          listItemName="Emissions Intensity Report"
         >
           <TaskListItem
             stepName="6"
             linkUrl={getProjectRevisionFormPageRoute(id, 6)}
+            formTitle="Emissions Intensity Report"
+            formStatus={quarterlyReportsStatus}
```

Comment:
should be `emissionsIntensityReportStatus` (or whatever it ends up being called)